src/backend/routers/case_manager.test.ts

```diff
@@ -277,9 +306,28 @@ test("addStudent - invalid email", async (t) => {
   }
 });
 
+test("addStudent - paras do not have access", async (t) => {
+  const { trpc } = await getTestServer(t, { authenticateAs: UserType.Para });
+
+  const error = await t.throwsAsync(async () => {
+    await trpc.case_manager.addStudent.mutate({
+      first_name: "Foo",
+      last_name: "Bar",
+      email: "invalid-email",
```

Comment:
Nitpick: this shouldn't be an invalid email because that's not what we're testing. I'm going to fix this